src/import/sst_importer.rs

```diff
@@ -88,19 +89,19 @@ impl SSTImporter {
     pub fn download(
         &self,
         meta: &SSTMeta,
-        url: &str,
+        backend: &StorageBackend,
         name: &str,
         rewrite_rule: &RewriteRule,
         speed_limiter: Option<Arc<IOLimiter>>,
     ) -> Result<Option<Range>> {
         debug!("download start";
             "meta" => ?meta,
-            "url" => url,
+            "url" => ?backend,
```

Comment:
The `backend` may contain secret data. I'm not sure if it's a good idea to log it.

Reply:
@tennix  how about just log the URL `s3://bucket/prefix/`? (we need to forward-port the patch to master)

Reply:
There is no url field, for debug purpose, I think all other parameters need to be logged. For the secret, I think we should print `<hidden-secret>` or whatever meaningful for non-empty secret, and `empty` for empty secret.

Reply:
the struct is generated by protobuf/prost so I don't think we can control its debug output (unless we use a custom formatter)